configure.ac

```diff
@@ -63,5 +63,9 @@ AM_CONDITIONAL([HAVE_PKG_CONFIG], [test "x$PKG_CONFIG" != "x"])
 AS_IF([test "x$PKG_CONFIG" != "x"], [
     AC_CONFIG_FILES([libuv.pc])
 ])
+AM_COND_IF([LINUX], [
+    AC_CHECK_HEADERS([sys/epoll.h], , [AC_MSG_ERROR([Could not find sys/epoll.h])])
+    AC_CHECK_FUNCS([epoll_pwait], , [AC_MSG_ERROR([Could not find epoll_pwait()])])
+])
```

Comment:
Can you remove the epoll_pwait check?  I'm not sure if it's useful to check for sys/epoll.h because of the GYP build (which doesn't do any configure time feature checks whatsoever.)